### PR TITLE
James 2493 : upgrade logging libraries

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by rouazana** [comment]
```
[42ea68ca4f4ef36ad92addb3176eef6e27895c66] Results :
[42ea68ca4f4ef36ad92addb3176eef6e27895c66] 
[42ea68ca4f4ef36ad92addb3176eef6e27895c66] Tests in error: 
[42ea68ca4f4ef36ad92addb3176eef6e27895c66]   CassandraConfigurationReadingTest.provideCassandraConfigurationShouldReturnDefaultOnEmptyConfigurationFile:33 NoClassDefFound
[42ea68ca4f4ef36ad92addb3176eef6e27895c66]   CassandraConfigurationReadingTest.provideCassandraConfigurationShouldReturnRightConfigurationFile:40 NoClassDefFound
[42ea68ca4f4ef36ad92addb3176eef6e27895c66]   QueryLoggerConfigurationTest.fromShouldNotThrowWithMinimalConfigAboutAConstantThresholdSlowQueryLogger:32 NoClassDefFound
[42ea68ca4f4ef36ad92addb3176eef6e27895c66]   QueryLoggerConfigurationTest.fromShouldNotThrowWithPersonalizedConfigAboutPercentileSlowQuerryLogger:41 NoClassDefFound
[42ea68ca4f4ef36ad92addb3176eef6e27895c66]   QueryLoggerConfigurationTest.fromShouldThrowIfConfigAboutLoggerIsInvalid:54 NoClassDefFound
```

**Comment by mbaechler** [comment]
```
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] Tests run: 104, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 5.812 sec <<< FAILURE! - in org.apache.james.mailbox.store.search.SimpleMessageSearchIndexTest
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] searchWithPDFAttachmentShouldReturnMailsWhenAttachmentContentMatches  Time elapsed: 0.066 sec  <<< ERROR!
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 	at org.apache.james.mailbox.store.search.AbstractMessageSearchIndexTest.searchWithPDFAttachmentShouldReturnMailsWhenAttachmentContentMatches(AbstractMessageSearchIndexTest.java:1346)
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] Caused by: java.lang.ClassNotFoundException: org.apache.commons.logging.LogFactory
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 	at org.apache.james.mailbox.store.search.AbstractMessageSearchIndexTest.searchWithPDFAttachmentShouldReturnMailsWhenAttachmentContentMatches(AbstractMessageSearchIndexTest.java:1346)
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] Running org.apache.james.mailbox.store.search.PDFTextExtractorTest
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.205 sec <<< FAILURE! - in org.apache.james.mailbox.store.search.PDFTextExtractorTest
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] extractContentShouldExtractPDF  Time elapsed: 0.015 sec  <<< ERROR!
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 	at org.apache.james.mailbox.store.search.PDFTextExtractorTest.extractContentShouldExtractPDF(PDFTextExtractorTest.java:70)
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] Caused by: java.lang.ClassNotFoundException: org.apache.commons.logging.LogFactory
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 	at org.apache.james.mailbox.store.search.PDFTextExtractorTest.extractContentShouldExtractPDF(PDFTextExtractorTest.java:70)
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] Results :
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] 
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b] Tests in error: 
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b]   PDFTextExtractorTest.extractContentShouldExtractPDF:70 » NoClassDefFound org/a...
[c6ed00c383fc8f46851a61ab1020c32c63ebae4b]   AbstractMessageSearchIndexTest.searchWithPDFAttachmentShouldReturnMailsWhenAttachmentContentMatches:1346 » NoClassDefFound
```

File: server/protocols/protocols-smtp/pom.xml [hunk 1]
```diff
@@ -174,11 +174,6 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
```
Comment (chibenwa) [inline server/protocols/protocols-smtp/pom.xml]: Don't you add a logback dependency here to keep logs in tests?

**Comment by chibenwa** [comment]
test this please (no build output)

**Comment by chibenwa** [comment]
```
[652b55c8c0a1539edca34d675ce113c21a4f450a] Running org.apache.james.mailbox.spring.SpringMailboxTest
[652b55c8c0a1539edca34d675ce113c21a4f450a] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.204 sec <<< FAILURE! - in org.apache.james.mailbox.spring.SpringMailboxTest
[652b55c8c0a1539edca34d675ce113c21a4f450a] org.apache.james.mailbox.spring.SpringMailboxTest  Time elapsed: 0.203 sec  <<< ERROR!
[652b55c8c0a1539edca34d675ce113c21a4f450a] java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory
[652b55c8c0a1539edca34d675ce113c21a4f450a] 	at org.apache.james.mailbox.spring.SpringMailboxTest.beforeClass(SpringMailboxTest.java:35)
[652b55c8c0a1539edca34d675ce113c21a4f450a] Caused by: java.lang.ClassNotFoundException: org.apache.commons.logging.LogFactory
[652b55c8c0a1539edca34d675ce113c21a4f450a] 	at org.apache.james.mailbox.spring.SpringMailboxTest.beforeClass(SpringMailboxTest.java:35)
[652b55c8c0a1539edca34d675ce113c21a4f450a] 
[652b55c8c0a1539edca34d675ce113c21a4f450a] 
[652b55c8c0a1539edca34d675ce113c21a4f450a] Results :
[652b55c8c0a1539edca34d675ce113c21a4f450a] 
[652b55c8c0a1539edca34d675ce113c21a4f450a] Tests in error: 
[652b55c8c0a1539edca34d675ce113c21a4f450a]   SpringMailboxTest.beforeClass:35 » NoClassDefFound org/apache/commons/logging/...
[652b55c8c0a1539edca34d675ce113c21a4f450a] 
[652b55c8c0a1539edca34d675ce113c21a4f450a] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by chibenwa** [comment]
test this please (with JAMES-2500)

**Comment by mbaechler** [comment]
```
[6787c0862ea9041b4efa2847d5fe1574480c9047] fromACLShouldFilterNegatedUsers  Time elapsed: 0.003 sec  <<< ERROR!
[6787c0862ea9041b4efa2847d5fe1574480c9047] java.lang.NoClassDefFoundError: Could not initialize class org.apache.james.jmap.model.mailbox.Rights
[6787c0862ea9041b4efa2847d5fe1574480c9047] 	at org.apache.james.jmap.model.mailbox.RightsTest.fromACLShouldFilterNegatedUsers(RightsTest.java:116)
[6787c0862ea9041b4efa2847d5fe1574480c9047] 
```

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**